traefik/templates/service-metrics.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if (and (.Values.metrics.prometheus.service).enabled (not .Values.hub.enabled)) -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-metrics
```

Comment:
You should use `traefik.fullname`. See service-internal.yaml for an example

Reply:
I did not see that one - c&p the logic from internal service into the metrics service.